content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -2,156 +2,88 @@
-
-For our example we need to define what accounts relates to our project's
-expenses. Create a position and add items to select the accounts.
-
-.. image:: budget/budget03.png
-   :align: center
+To define budgetary positions go to :menuselection:`Accounting --> Configuration -->
+Budgetary positions` and :guilabel:`Create`. Add a :guilabel:`Name` to your budgetary position.
+Each budgetary position can have any number of accounts from the chart of accounts, though it must
+have at least one.
 
-In this case we select the three relevant accounts used wherein we will
-book our expenses.
-
-.. image:: budget/budget04.png
-   :align: center
+Use case
+========
 
-Click on *Select*.
+Let’s illustrate this with an example: we just started a project with Smith & Co, and we would like
+to budget the income and expenses of that project.
 
-.. image:: budget/budget05.png
-   :align: center
+We plan to have a revenue of 1000, and we don’t want to spend more than 700.
 
-Save the changes to confirm your Budgetary position.
+First, we need to define what accounts relate to our project’s expenses. Go to
+:menuselection:`Accounting --> Configuration --> Budgetary positions` and :guilabel:`Create` a
+position and add the accounts wherein we will book our expenses.
 
-Repeat this steps to create a revenue budgetary position. Only in this
-case select the relevant income accounts.
+.. image::  budget/smith-and-co-expenses.png
+  :align: center
 
-Analytical account
-------------------
+Now, let's repeat the steps to create a budgetary position that reflects the revenue.
 
-Odoo needs to know which costs or
-expenses are relevant to a specified budget. To do so we need to link
-our invoices and expenses to a defined analytical account. Create an
-analytical account by entering the Accounting module and clicking
-:menuselection:`Advisers --> Analytic Accounts --> Open Charts`. Create a new Account
-called Smith&Co project and select the related partner.
+.. image::  budget/smith-and-co-revenue.png
+  :align: center
 
-.. image:: budget/budget06.png
-   :align: center
-
-Set a budget
-============
+Analytical accounts
+-------------------
 
-Let's now set our targets for our budget. We specified that we expect to
-gain 1000 with this project and we would like not to spend more than
-700.
+Odoo needs to know which costs or expenses are relevant to a specified budget because the above
+general accounts may be used for different projects. To do so we need to link our invoices and
+expenses to a defined analytical account: Go to :menuselection:`Accounting --> Configuration -->
+Analytic Accounts` and :guilabel:`Create` a new :guilabel:`Analytic Account` called Smith & Co.
 
-To set those targets, enter the accounting app, select :menuselection:`Advisers -->
-Budgets` and create a new Budget.
+Each budgetary position can have any number of accounts from the general ledger (the main chart of
+accounts) assigned to it, though it must have at least one.
 
```

Comment:
**Important**: I cannot leave the page without specifying a "plan". Also, this might be relevant in a future comment (regarding purchase lines)

Reply:
Still nothing said about plan, but it's important since you cannot leave without specifying a plan in Odoo 16 (not necessary in previous version,s though)

Reply:
Also, I think you forgot to remove that sentence as it as not link to what is explain before and because it is said earlier when you talk about budgetory positions